src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -31,6 +31,7 @@ internal class LowLevelClientWriter : ClientWriter
     {
         private static readonly CSharpType RequestContextParameterType = new(typeof(RequestContext), true);
 
+        private static readonly Parameter WaitForCompletionParameter = new("waitForCompletion", "true if method should return after the long-running operation is completed; otherwise, false.", new CSharpType(typeof(bool)), null, false);
```

Comment:
@annelo-msft , can you think of a better description here?

Reply:
How about this?

`true` if the method should wait to return until the long-running operation has completed on the service; `false` if it should return after starting the operation.  For more information on long-running operations, please see [Azure.Core Long-Running Operation samples.](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md).